src/post.pegjs

```diff
@@ -88,6 +90,22 @@ HTML_Comment
     type: "HTML_Comment",
     innerText: cs.join('')
   } }
+
+HTML_Tag_Void
+  = t:HTML_Tag_Open
+  & { return undefined !== {
+      'br': true,
+      'col': true,
+      'embed': true,
+      'hr': true,
+      'img': true,
+      'input': true
```

Comment:
I wonder, is the list of these elements described in a specification? Because IIRC `link` and `meta` are two more options which don't require an explicit close.